defender/wallet-migrator/serverless.yml

```
@@ -0,0 +1,56 @@
+service: wallet-migrator
+configValidationMode: error
+frameworkVersion: "3"
+
+provider:
+  name: defender
+  stage: ${opt:stage, 'dev'}
+  stackName: wallet_migrator
+  ssot: false
+
+# Configure and rename sample.secrets.yml to .secrets/dev.yml
+custom:
+  secrets: ${file(../.secrets/${self:provider.stage}.yml)}
+  config: ${file(./config.${self:provider.stage}.yml)}
+
+# Make sure that you have created keys in your Defender account and added them to your .secrets/dev.yml file
+defender:
+  key: ${self:custom.secrets.keys.defender-api-key}
+  secret: ${self:custom.secrets.keys.defender-api-secret}
+
+functions:
+  autotask-1:
+    name: Wallet Migrator Autotask
+    path: autotask-1
+    trigger:
+      type: schedule
+      # set to never run on a schedule and only run manually
+      frequency: 1000000000
+    paused: false
```

```suggestion
    paused: true
```
Autotasks are known to randomly trigger